emails/templates/private-cloud/AdminCreateRequest.tsx

```diff
@@ -7,15 +7,13 @@ import { Tailwind } from '@react-email/tailwind';
 import NamespaceDetails from '../../components/NamespaceDetails';
 import Closing from '../../components/Closing';
 import { TailwindConfig } from '../../components/TailwindConfig';
-const defaultTheme = require('tailwindcss/defaultTheme');
 
 interface EmailProp {
   request: PrivateCloudRequestWithRequestedProject;
 }
 
 export const NewRequestTemplate = ({ request }: EmailProp) => {
   if (!request) return <></>;
-
```

Comment:
to reduce the unnecessary code changes and also for readability, let's keep this new line.